src/utils/event.ts

```diff
@@ -181,15 +181,33 @@ export const identifyEvent = async (event: UnidentifiedEvent): Promise<UnsignedE
   return { ...event, id }
 }
 
-export function getRelayPrivateKey(relayUrl: string): string {
+let privateKeyCache: string | undefined
+export function getRelayPrivateKey(secret: string): string {
```

Comment:
Is `secret` an optional parameter? If so we could make the type indicate this

Reply:
it should always be the relay's URL, but it is ignored when `RELAY_PRIVATE_KEY` env var is set

Reply:
Ok, why is the param called `secret`?